lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -227,6 +230,10 @@ def instructor_dashboard_2(request, course_id):
 
     certificate_invalidations = CertificateInvalidation.get_certificate_invalidations(course_key)
 
+    # x block asides enabled, included rapid response tab
+    if XBlockAsidesConfig.current().enabled:
+        sections.append(_section_rapid_response(course_key))
```

Comment:
Have you tested this with the XBlockAsidesConfig option set to false? Just want to make sure

Reply:
Yes, I have.